src/qinst/instrument.py

```diff
@@ -56,3 +58,18 @@ def get(self, *args) -> dict:
             else:
                 raise RuntimeError(f"The instrument does not have the {key} parameter.")
         return values
+
+    def validate_opt(self, opt: str, allowed: tuple):
```

Comment:
Since `self` is not used, this can be a staticmethod:
```suggestion
    @staticmethod
    def validate_opt(self, opt: str, allowed: tuple):
```